StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/SheetNavigationBar.swift

```diff
@@ -132,16 +132,19 @@ class SheetNavigationBar: UIView {
     // MARK: -
     enum Style {
         case close(showAdditionalButton: Bool)
-        case back
+        case back(showAdditionalButton: Bool)
```

Comment:
I don't feel too strongly about this, I see this more as a convince to toggle the additional button when switching styles, instead of switching the style then hiding/showing the additional button if needed

Reply:
For convenience we could still have a `update(style: Style, showAdditionalButton: Bool)` method. But it's very minor so I'm fine to leave it alone.